config/crd/bases/skupper_secured_access_crd.yaml

```diff
@@ -10,7 +10,14 @@ spec:
       storage: true
       schema:
         openAPIV3Schema:
-          description: "An internal resource used to create secure access to pods"
+          description: |-
+            SecuredAccess is an internal resource used to create secure access to pods.
```

Comment:
Internal resource for secure access to the site router and grant server. See RouterAccess for user configuration options.

Reply:
Sorry! Did not love this either
- would prefer leaving it open ended: used for routers and grants servers today, but would get reused for future features if needed.
- referring users to RouterAccess config also seems off: as if they RouterAccess options were applicable to SecuredAccess (they mostly line up quite well). Also we have no good controller docs to point to for grants server configuration options.